README.md

```diff
@@ -16,7 +16,7 @@ To run the site locally with Docker, use the following command from the toplevel
 
 ```bash
 # First time: (slow)
-docker run --name istio-jekyll --volume=$(pwd):/srv/jekyll  -it -p 127.0.0.1:4000:4000 jekyll/jekyll:3.5.2 sh -c "bundle install && rake test && jekyll serve --incremental"
+docker run --name istio-jekyll --volume=$(pwd):/srv/jekyll  -it -p 4000:4000 jekyll/jekyll:3.5.2 sh -c "bundle install && rake test && bundle exec jekyll serve --incremental --host 0.0.0.0"
```

Comment:
much simpler, thanks

qq: I don't see --host on https://docs.docker.com/engine/reference/run/ ?

Reply:
it's for `jekyll serve`

Reply:
oh duh me ! it's inside the -c ""  !